<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QToolBox" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>URL Inspector Automator by RankSense</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="Configuration">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>457</height>
    </rect>
   </property>
   <attribute name="label">
    <string>Configuration</string>
   </attribute>
   <widget class="QLineEdit" name="userFolder">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>./chrome-dev</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User folder:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="website">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>https://www.marcopromos.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Website to check:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay between checks:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="delay">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>8</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>seconds</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="urls2Check">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>491</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="plainText">
     <string>/
/category/24-hour-promotional-items
/Product/FRN-DELEGATE/H1-2662
/category/24-hour-quick-ship-bags</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>URLs to check</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Element selectors:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectorsConf">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>./selectors.ini</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Launch Chrome Debug Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inspect URLs</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wsURI">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>WS URI:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export to CSV</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="csvFile">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>./inspected.csv</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Export file:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>URLs not indexed</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="urlsNotIndexed">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>390</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Not indexed criteria:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="notIndexCriteria">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>390</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alternate page with proper canonical tag</string>
    </property>
   </widget>
   <widget class="QComboBox" name="actionBox">
    <property name="geometry">
     <rect>
      <x>833</x>
      <y>40</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Do Nothing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Test Live Not Indexed URLs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Submit Not Indexed URLs</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QWidget" name="Results">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>457</height>
    </rect>
   </property>
   <attribute name="label">
    <string>Results</string>
   </attribute>
   <widget class="QTableWidget" name="resultsWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1151</width>
      <height>401</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
